tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -261,9 +263,6 @@ void generateTileBitmap( Tile tile ) {
   }
 
   void addTileToCurrentTileCanvasView( Tile tile ) {
-    if( mTilesAlreadyRendered.contains( tile ) ) {
-      return;
-    }
```

Comment:
Assuming you're saying it's not needed anymore because we remove them all later, I think this is still a good idea to keep, so there's less traversal when removal happens (and, semantically, the tile shouldn't be added if it's already rendered).  Finally, the more atomic we can keep each PR the better - if we want to optimize later, no problem, but let's make sure we're only touching moving parts we're 100% sure about.